### PR TITLE
Add an ability to set <query-expression> through the vscode settings

### PR DESCRIPTION
**Pull request by maximMalofeev**
Add an ability to set <query-expression> through the vscode settings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -147,6 +147,11 @@
                     "uniqueItems": true,
                     "default": [],
                     "markdownDescription": "A list of Bazel command line options to use when building or running tests. From the [common options](https://docs.bazel.build/versions/master/command-line-reference.html#options-common-to-all-commands) and [build options](https://docs.bazel.build/versions/master/command-line-reference.html#build) (`test` honors all `build` options). One option per entry, no shell escaping is needed."
+                },
+                "bazel.commandLine.queryExpression": {
+                    "type": "string",
+                    "default": "...:*",
+                    "description": "A query language expression[https://bazel.build/query/language]. If the default option that inspects all the workspace doesn't fit, you could narrow it, for example: //part/you/want/...:*"
```
Comment (jfirebaugh) [inline package.json:154]:
```suggestion
                    "description": "A [query language expression](https://bazel.build/query/language) which determines the packages displayed in the workspace tree and quick picker. The default inspects the entire workspace, but you could narrow it. For example: `//part/you/want/...:*`"
```

**Review by jfirebaugh — COMMENTED** [review]
Thanks for the contribution! Just one documentation nit.

**Review by jfirebaugh — APPROVED** [review]

**Comment by maximMalofeev** [comment]
@jfirebaugh Thank you for the approve. Unfortunately cla/google check failed. Could you accept CLA? Or could you offer another way to complete this pull request?

**Comment by jfirebaugh** [comment]
The `cla/google` check is failing because the suggested change picked up my personal address as coauthor, which is not covered by the CLA, rather than `jfirebaugh@figma.com`, which is covered. Apparently GitHub does not have a way to choose the email address associated with a "suggested change"-generated commit, other than by changing the primary email address of your account.

@maximMalofeev, if you want to amend the commit to remove my coauthorship, that's fine. Or if you grant me permission to push changes to this PR, I can do it.

**Comment by maximMalofeev** [comment]
Done. Just checked Allow edits by maintainers option

**Pull request merged**